src/app/devices/devices.service.ts

```diff
@@ -256,7 +238,7 @@ export class DevicesService {
   }
 
   getStats (): Observable<DeviceStats> {
-    return this.http.get<DeviceStats>(`${environment.mpsServer}/devices/stats`)
+    return this.http.get<DeviceStats>(`${environment.mpsServer}/api/v1/devices/stats`)
```

Comment:
I think moving the /api/v1/ to be as part of the mpsServer baseaddress might be a better move?